File: semicolon/python codes/cornFlakes/rai shaGUL/NumberCounter.java
import java.util.Scanner;

public class NumberCounter {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        int positiveCount = 0;
        int negativeCount = 0;
        String choice;
        do {
            System.out.print("Enter a number: ");
            int num = scanner.nextInt();
            
         
            if (num > 0) {
                positiveCount++;
            } else if (num < 0) {
                negativeCount++;
            }
            
            System.out.print("Do you want to enter another number? (y/n): ");
            choice = scanner.next();
        } while (choice.equalsIgnoreCase("y") || choice.equalsIgnoreCase("Y"));
        
       
        System.out.println("Count of positive numbers: " + positiveCount);
        System.out.println("Count of negative numbers: " + negativeCount);
        
    }
}

File: semicolon/python codes/cornFlakes/rai shaGUL/number_counter.py
positive_count = 0
negative_count = 0

    
while True:
 try:
   num = int(input("Enter a number: "))
   if num > 0:
    positive_count += 1
   elif num < 0:
    negative_count += 1
 except ValueError:
        print("Invalid input. Please enter a valid integer.")

choice = input("Do you want to enter another number? (y/n): ")
if choice.lower() != 'y':
    break

print(f"Count of positive numbers: {positive_count}")
print(f"Count of negative numbers: {negative_count}")